extensions/vscode/src/granite/utils/compatibilityUtils.ts

```diff
@@ -2,7 +2,7 @@ import * as vscode from "vscode";
 
 import type { VsCodeWebviewProtocol } from "../../webviewProtocol";
 
-const incompatibleExtensionIds = new Set(["Continue.continue"]);
+const incompatibleExtensionIds = new Set(["redhat.granitecode"]);
```

Comment:
const incompatibleExtensionIds = new Set(["Continue.continue", "redhat.granitecode"]);

Reply:
we will still need to update the build script so it does not transform this particular file